build.gradle

```diff
@@ -41,9 +41,5 @@ dependencies {
         exclude group:'com.ibm.icu', module:'icu4j'
     }
     implementation 'org.antlr:antlr4-intellij-adaptor:0.1'
-    testCompile group: 'junit', name: 'junit', version: '4.11'
+    testImplementation group: 'junit', name: 'junit', version: '4.11'
 }
-
-generateGrammarSource {
-    arguments += ["-package", "org.antlr.jetbrains.sample.parser", "-Xexact-output-dir"]
-}
```

Comment:
Odd stuff. Replaced with a single package header in the grammar (not perfect either) and now the files are generated in the correct package with the correct package statement.

Reply:
I think there's a downside to this method, but I can't remember what it is 🤔 .

Reply:
The upside is that you can open the generated files in an IDE and not get compile errors :-)
I'm happy to revert it though, just say the word.